app/src/main/java/at/aau/edu/wizards/gameModel/GameModelCard.kt

```diff
@@ -27,6 +27,7 @@ sealed class GameModelCard {
         object Red : Color()
     }
 
+    // FIXME refactor for clarity, its always type+color, magic constants
```

Comment:
Please expand, I don't quite understand what you want. Are you not happy with the char representation?

Reply:
The messaging (representation and parsing) needs to be more self-explanatory. Please better describe it in code, e.g., dedicated DTOs and/or clear parsing logic without magic constants.

Reply:
done